src/types.jl

```diff
@@ -14,7 +14,7 @@ function linttype(ex::Expr, ctx::LintContext)
             adt = sube.args[i]
             if isa(adt, Symbol)
                 foundobj = lookup(ctx, adt)
-                if !isnull(foundobj) && get(foundobj).typeactual <: Type
+                if foundobj ≠ nothing && foundobj.typeactual <: Type
```

Comment:
I'd recommend `!==` for `nothing` because it avoids possible problems with weird `foundobj`, which might overload `==` in a strange way. Not a big deal though.